### PR TITLE
Fix linux pre commit

### PR DESCRIPTION
**Pull request by DavidFair**
*Marking High Priority, as we currently have PRs going through formatting CI effectively untested*

**Description of work.**
Fixes the git commit hook which was only checking the first file due to `$1` instead of `$@` to pass all args

**To test:**
- Checkout PR
- `pre-commit run clang-format --all-files` previously took <1s, it should now take at least >10s
- Try changing a random file and re-running with the all-files flag. It should now be picked up

~If you are happy with this could you ping me to get the corresponding [PR](https://github.com/mantidproject/pre-commit-hooks/pull/8/commits/4a3e150e80aaa0f2810af279279862817547e1a2) merged, then update the hook SHA / location~
Edit: merged

There is no associated issue. - Noticed when my PRs kept randomly breaking :frowning_face: 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by darominski — CHANGES_REQUESTED** [review]
Works as advertised. There's only one thing: when I run the pre-commit on all files, there are about 11 additional files in the  `qt/widgets/instrumentview/inc/MantidQtWidgets` directory, namely: `BankRenderingHelpers`, `BankRenderingHelpers`, `BinDialog`, `InstrumentTreeWidget`, `InstrumentWidgetEncoder`, `InstrumentWidgetTab`, `MiniPlotMpl`, `PeakMarker2D`, `PeakOverlay`, `PlotFitAnalysisPaneModel`, and `PlotFitAnalysisPanePresenter`. 
I think it's worth fixing them as well as those that you have already included to ensure the pre-commit hook works well after this is merged.

**Comment by martyngigg** [comment]
David is out today so I've pushed the fixed for the hook on top of his branch.

**Review by darominski — APPROVED** [review]
Thank you for the changes, I am happy to not keep it on hold for any longer.

**Pull request merged**
